config/vufind/SierraRest.ini

```diff
@@ -211,6 +211,13 @@ title_hold_excluded_item_codes = "e"
 ;a = patron_status_address_missing
 ;b = patron_status_card_blocked
 
+; This section allows modification of the default mapping from Sierra fine types to
+; VuFind fine types. By default fine types are displayed as is. Examples are listed
+; below.
+[FineTypeMappings]
+;Manual = ""
+;Overdue Renewal = "Overdue"
```

Comment:
Would it make sense to uncomment this by default to retain the historical behavior, or is there a reason we no longer want that? (No strong feelings about the matter -- just asking).

Reply:
I opted to remove the defaults since they came from us and the changes were requested by the Sierra library. Also hiding the "Manual" type no longer makes so much sense now that it has its own column.

Reply:
And for the "Overdue Renewal" I've no recollection why it was done in the first place since similar mappings are not used anywhere else, and the library specifically asked not to do that since you lose some information that might be of value.